tests/util_tests/test_qt_importer.py

```diff
@@ -160,7 +157,6 @@ def test_qt_importer_with_pyside6_interface_qt4(self):
         assert qt.base["__version__"] is qt.binding_version
 
     @skip_if_pyside6(found=False)
-    @skip_if_pyqt4(found=True)
     @skip_if_pyside(found=True)
```

Comment:
What about this one? Should we remove it too?

Reply:
yeah, we should. I'm still working on tests